optics-core/src/Optics.hs

```diff
@@ -1,29 +1,79 @@
+-- |
+--
+-- Module: Optics
+-- Description: The main module, usually you only need to import this one.
```

Comment:
description is visible in the module listing. Highlighting this one will make people click it. Hopefully. (Hackage and Stackage may render the module listing differently though).